input/parameters/lithium-ion/anodes/graphite_mcmb2528_Marquis2019/parameters.csv

```diff
@@ -11,6 +11,7 @@ Negative electrode OCP [V],[function]graphite_mcmb2528_ocp_Dualfoil1998,
 Negative electrode porosity,0.3,Scott Moura FastDFN,electrolyte volume fraction
 Negative electrode active material volume fraction,0.7,,assuming zero binder volume fraction
 Negative particle radius [m],1E-05,Scott Moura FastDFN,
+Negative particle distribution,1,,
```

Comment:
is it worth having a comment here just to make it clear that this is a size distribution in x, rather than a distribution of particle sizes at each x?